server/game/cards/characters/01/thequeenofthorns.js

```diff
@@ -6,24 +6,17 @@ class TheQueenOfThorns extends DrawCard {
             when: {
                 afterChallenge: (event, challenge) => challenge.winner === this.controller && challenge.isParticipating(this) && challenge.challengeType === 'intrigue'
             },
-            handler: () => {
-                this.game.promptForSelect(this.controller, {
-                    activePromptTitle: 'Select a character to put in play',
-                    source: this,
-                    cardCondition: card => card.location === 'hand' && card.getCost() && card.getType() === 'character' && card.isFaction('tyrell'),
-                    onSelect: (p, card) => this.onCardSelected(p, card)
-                });
+            target: {
+                activePromptTitle: 'Select a character to put into play',
+                cardCondition: card => card.location === 'hand' && card.controller === this.controller &&
```

Comment:
I'm not 100% sure, but wouldn't that allow you to target a card in an opponents hand (provided you happen to click one that pass the other conditions)? Or is that the opponent's hand not a legal target area anyway?